tests/data/PersonRegGroupDefs.xml

```diff
@@ -1,1086 +1,1508 @@
+    <gname>UW Student Employees</gname>
+    <desc>UW student employees.  This group is automatically maintained in near-real time from UW-IT Identity Registry data. Includes a two week grace period for recent terminees.</desc>
+    <contact>iam-support</contact>
+    <classification>restricted</classification>
+    <memberViewers>
+      <viewer>cn="uw_employee"</viewer>
+      <viewer>cn="u_groups_affiliation_employee-read"</viewer>
+    </memberViewers>
+    <memberUpdaters>
+      <updater>dns="iam-tools-irws-watcher.cac.washington.edu"</updater>
+    </memberUpdaters>
+     <realtime>
+       <filter type="1">
+        <category-status>
+          <value>14-1</value>
+          <value>14-2</value>
+        </category-status>
+       </filter>
+     </realtime>
+    <query>
+SELECT distinct u.uwnetid 
+FROM uwnetid u 
+JOIN category c
+  on u.validation_id = c.validation_id
+  and u.status_code = 30
+  and c.category_code = 14
+  and c.status_code != 3
+ORDER by u.uwnetid;</query>
+  </group>
+
+  <group cn="uw_affiliation_affiliate-employee">
+    <title>UW Affiliate Employees</title>
+    <gname>UW Affiliate Employees</gname>
+    <desc>Contingent Workers in Workday. This group is automatically maintained in near-real time from UW-IT Identity Registry data. Includes a two week grace period for recent terminees.</desc>
+    <contact>iam-support</contact>
+    <classification>public</classification>
+    <memberViewers>
+      <viewer>cn="u_groups_affiliation_employee-read"</viewer>
+      <viewer>cn="uw_employee"</viewer>
+    </memberViewers>
+    <memberUpdaters>
+      <updater>dns="iam-tools-irws-watcher.cac.washington.edu"</updater>
+    </memberUpdaters>
+     <realtime>
+       <filter type="1">
+        <category-status>
+          <value>15-1</value>
+          <value>15-2</value>
+        </category-status>
+       </filter>
+     </realtime>
+    <query>
+SELECT distinct u.uwnetid 
+FROM uwnetid u 
+JOIN category c
+  on u.validation_id = c.validation_id
+  and u.status_code = 30
+  and c.category_code = 15
+  and c.status_code != 3
+ORDER by u.uwnetid;</query>
+  </group>
+
+  <group cn="uw_affiliation_staff-non-uwm-workforce">
+    <title>UW Staff Employees not identified as UW Medicine Workforce</title>
+    <gname>UW Staff Employees not identified as UW Medicine Workforce</gname>
+    <desc>Employees with a staff position in Workday who are not identified as UW Medicine Workforce. This group is updated daily from UW-IT Identity Registry data.</desc>
+    <classification>restricted</classification>
+    <memberViewers>
+      <viewer>cn="uw_employee"</viewer>
+      <viewer>cn="u_groups_affiliation_employee-read"</viewer>
+    </memberViewers>
+    <memberUpdaters>
+      <updater>dns="iam-tools-irws-watcher.cac.washington.edu"</updater>
+    </memberUpdaters>
+    <contact>iam-support</contact>
+     <realtime>
+       <filter type="1">
+       <category-status>
+          <value>4-50</value>
+          <value>14-50</value>
```

Comment:
1. Should these `50`s be `30`s?
2. Should there be a `<value>288-30</value>` (or 288-50?)
3. Should anything about category_code 13 be included here?

Reply:
The irws-watcher software doesn't know how to handle "not" and I think I was afraid anyways that including <realtime/> would cause users to be removed from the group because they didn't match the criteria it knows how to read here. I'm going to pull this group config out of this repo just to be safe. 

This group has been and will continue to be updated only once per day (UWMed still doesn't have 2fa required so this group is still needed but I'll suggest it be deleted when that situtation changes.